### DISCUSSION TITLE
Need a little startup help

### DISCUSSION THREAD

**Discussion by Ricks88** [PiotrMachowski/Home-Assistant-custom-components-Custom-Templates#9] [Q&A]
Hello

I would like to ask some help to get this working.
I'm making use of packages in my configuration.yaml, like:
"packages: !include_dir_named integrations" 
in there I have for example: "template: !include_dir_list ../entities/template", where I put all templates.

How do I do this for
```yaml
custom_templates:
  preload_translations:
    - en
    - nl
```

Should I link to the custom_templates folder?
I can not link this directly in my configuration.yaml

And where to I put the files to start with the entities I want to translate? Should they be .yaml?

Thanks!

**Comment by PiotrMachowski** [comment]
Once you have added this config to `configuration.yaml` you can use new functions in normal template sensors.

**Reply by Ricks88** [reply]
do I have to make separate template sensors for each entity I want to translate or can I do that directly in lovelace for each sensor?

**Reply by Ricks88** [reply]
because it works in devtools but I don't know how to actually show it there in lovelace

![devtools](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/91467918/e835b588-19b5-4b65-ad2e-c93b7c81cc2c)

`type: custom:button-card
entity: sun.sun
aspect_ratio: 0
.......
name: sunsun test`

![test](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/91467918/b269c791-88c0-4962-9db2-aef811620b46)

**Reply by PiotrMachowski** [reply]
If the card you want to use supports templates you can use it directly, but generally speaking you have to create a template sensor. Buuuut lovelace cards should use translated values out of the box

**Reply by Ricks88** [reply]
Ow sorry , did not read this one yet.

Then i better create template sensors, so they stay translated forever haha.

"Buuuut lovelace cards should use translated values out of the box", Yes, it did long time ago. i have no translation anymore in button cards. Do not understand why.

![waterbutton](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/91467918/393ab1d5-353c-4c44-b41a-5eec569edcea)
![waterinfo](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/assets/91467918/6bbfd46f-1dd1-43f1-89c2-c919f44da991)

"Off" should be "Droog"

**Comment by Ricks88** [comment]
@PiotrMachowski 

Sorry, but can not get it to work:

```yaml
platform: template

sensors:
  sun_nl:
    friendly_name: "Sun NL"
    value_template: "{{ 'sun.sun' | ct_state_translated('nl') }}"
```

 No filter named 'ct_state_translated'.) for dictionary value @ data['sensors']['sun_nl']['value_template']. Got "{{ 'sun.sun' | ct_state_translated('nl') }}".

**Reply by PiotrMachowski** [reply]
Have you added appropriate config in `configuration.yaml`?

**Reply by Ricks88** [reply]
Yes i have

**Reply by PiotrMachowski** [reply]
Have you restarted HA after adding `custom_templates` in `configuration.yaml`?

**Reply by Ricks88** [reply]
Yes, always

**Reply by Ricks88** [reply]
but i use   packages: !include_dir_named integrations
and 
custom_templates:
  preload_translations:
    - en
    - nl

in a separate file in the integrations map

**Reply by PiotrMachowski** [reply]
Have you tried to add it in `configuration.yaml`?
